Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM registry.redhat.io/ubi8/ubi-minimal
 WORKDIR /app-root/
-RUN microdnf install -y python36 python3-devel curl python3-pip git tar xz bzip2 unzip && \
+RUN microdnf install -y python38 python38-devel curl python3-pip git tar xz bzip2 unzip && \
```

Comment:
I still have few concern related to how puptoo image is build. Like - 
1. Do we really need python38 here? because in pyproject.toml we have specified `~3.6` and while running in prod it will uses python3.6. 
2. Poetry document says it requires python 3.7+ [https://python-poetry.org/docs/](https://python-poetry.org/docs/) 

But I think we can address these things in separate PR and have separate Jira ticket for this.